DOCUMENTATION.md

```diff
@@ -325,145 +322,143 @@ The `package` property is an array list of packages to be outputted. If the prop
-| dmg          | -                  | :white_check_mark: | -                                     |
-| mas          | -                  | :white_check_mark: | -                                     |
-| mas-dev      | :white_check_mark: | -                  | -                                     |
-| pkg          | -                  | :white_check_mark: | -                                     |
-| 7z           | :white_check_mark: | :white_check_mark: | :white_check_mark:                    |
-| zip          | :white_check_mark: | :white_check_mark: | :white_check_mark:                    |
-| tar.xz       | :white_check_mark: | :white_check_mark: | :white_check_mark:                    |
-| tar.lz       | :white_check_mark: | :white_check_mark: | :white_check_mark:                    |
-| tar.gz       | :white_check_mark: | :white_check_mark: | :white_check_mark:                    |
-| tar.bz2      | :white_check_mark: | :white_check_mark: | :white_check_mark:                    |
-| dir          | :white_check_mark: | :white_check_mark: | :white_check_mark:                    |
-| nsis         | -                  | -                  | :white_check_mark:                    |
-| nsis-web     | -                  | -                  | :white_check_mark:                    |
-| portable     | -                  | -                  | :white_check_mark:                    |
-| appx         | -                  | -                  | :white_check_mark: <sup>**[1]**</sup> |
-| msi          | -                  | -                  | :white_check_mark:                    |
-| AppImage     | :white_check_mark: | -                  | -                                     |
-| snap         | :white_check_mark: | -                  | -                                     |
-| deb          | :white_check_mark: | -                  | -                                     |
-| rpm          | :white_check_mark: | -                  | -                                     |
-| freebsd      | :white_check_mark: | -                  | -                                     |
-| pacman       | :white_check_mark: | -                  | -                                     |
-| p5p          | :white_check_mark: | -                  | -                                     |
-| apk          | :white_check_mark: | -                  | -                                     |
-
-* **[1]** Only Window 10 can build AppX packages.
-
-#### Setting the package `arch`
+| Package Type |  Linux   |      macOS       |           Windows           |
+| ------------ | :------: | :--------------: | :-------------------------: |
+| default      |    -     | `dmg`<br />`zip` |              -              |
+| dmg          |    -     |     &#10004;     |              -              |
+| mas          |    -     |     &#10004;     |              -              |
+| mas-dev      | &#10004; |        -         |              -              |
```

Comment:
Also, consider `&#9989;` (&#9989;) I believe it's a bit more distinctive than just a dull check mark.